windows_install.ps1

```diff
@@ -0,0 +1,48 @@
+#Requires -Version 5
```

Comment:
Can we move this file and `install.sh` under another folder? It will make the repo a bit cleaner.
maybe `Dockerfile`,  and `.goreleaser.yml` as well if it's possible.

Reply:
`Dockerfile` and `.goreleaser.yml` are located in root folder in purpose. That's where they are expected to be (its a convention with all the tools that use them)
I'll move the install scripts into `scripts` folder